dffml/high_level/ml.py

```diff
@@ -140,6 +159,17 @@ async def accuracy(
             # through something like pydantic. See issue #36
             raise TypeError(f"{accuracy_scorer} is not an AccuracyScorer")
         # Run accuracy method
+        # if accuracy scorer expects predict feature(s), pass as argument else don't
+        if (
+            accuracy_scorer.__class__.__qualname__
+            not in NO_PREDICT_FEATURE_SCORERS
```

Comment:
The main module really shouldn't have to know anything about plugins, let's remove `NO_PREDICT_FEATURE_SCORERS`.

Reply:
Why was this logic added?

Reply:
To accommodate spacy ner but it seems like it doesn't have python api.